arboretum/permissions/fetchers/github/fetch_org_collaborators.py

```diff
@@ -23,7 +23,15 @@
 from compliance.utils.data_parse import get_sha256_hash
 from compliance.utils.services.github import Github
 
+from deprecated import deprecated
 
+
+@deprecated(
+    reason=(
+        'Github org integrity collaborators fetcher functionality'
+        'moved to Github org integrity permissions fetcher'
+    )
+)
```

Comment:
Let's remove all of this.  We shouldn't deprecate the fetcher since it is the parent class for the new org integrity fetcher.  Sorry for causing the unnecessary thrash here.